src/integration/common/radioSchedule.js

```diff
@@ -1,7 +1,7 @@
 const servicesWithRadioSchedule = ['indonesia'];
 const servicesWithoutRadioSchedule = ['gahuza'];
 
-export default ({ isAmp }) => {
+export default ({ isAmp = false } = {}) => {
```

Comment:
Did we need the extra `= {}`
```suggestion
export default ({ isAmp = false }) => {
```

Reply:
If I do that I get the error 'TypeError: Cannot read properties of undefined (reading 'isAmp')'.
Having the extra = {} stops this error.

Reply:
Apparently this is because {isAmp = false} as the default value then expects an object being passed in. If we pass in nothing with radioSchedule(), then it is getting 'undefined' which is trying to be destructured, but cannot as it isn't an object. Adding = {} provides an empty object as the default value for the entire parameter, so when nothing is provided in radioSchedule() call it defaults to an empty object, which then when destructuring happens it becomes the default false, so does not error.

Reply:
Ahhhh I see. It probably doesn't need to be a destructed object anymore (i.e. it could be a single param with a default value), but let's leave it for now, thanks for checking! 👍